docs/sanic/websocket.rst

```diff
@@ -51,5 +51,6 @@ You could setup your own WebSocket configuration through ``app.config``, like
     app.config.WEBSOCKET_MAX_QUEUE = 32
     app.config.WEBSOCKET_READ_LIMIT = 2 ** 16
     app.config.WEBSOCKET_WRITE_LIMIT = 2 ** 16
+    app.config.WEBSOCKET_PING_INTERVAL = 20
 
```

Comment:
`WEBSOCKET_PING_TIMEOUT`?

Reply:
Also, it is probably worth adding in a line in the notes here that these setting will have no impact if running in ASGI mode.

Reply:
Have updated the docs as suggested.